Frends.Excel.Parse/Frends.Excel.Parse/Definitions/Result.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel;
+using System.Data;
+
+namespace Frends.Excel.Parse.Definitions;
+
+public class Result
+{
+    /// <summary>
+    /// False if conversion fails.
+    /// </summary>
+    [DefaultValue("false")]
+    public bool Success { get; set; }
```

Comment:
Should the setter be `private set`?

Reply:
Yes. And I actually opt for internal set today, so that I can freely use this field inside the assembly, but outsides can't